torch/testing/_internal/distributed/rpc/dist_autograd_test.py

```diff
@@ -2525,25 +2531,44 @@ def test_thread_local_context_id(self):
 
 
 class CudaDistAutogradTest(CommonDistAutogradTest):
-    @skip_if_lt_x_gpu(1)
-    @dist_init
-    def test_gpu_simple(self):
-        t1 = torch.rand(3, 3, requires_grad=True, device="cuda:0")
-        t2 = torch.rand(3, 3, requires_grad=True, device="cuda:0")
-        (t1 + t2).sum().backward()
+
+    def _gpu_simple(self, t1, t2, sparse):
```

Comment:
`sparse` is redundant as `t1.is_sparse()` can also tell that?

Reply:
Yes, I will update the file.